vignettes/userguide.Rmd

```diff
@@ -285,6 +285,11 @@ time after you restart R.
       otherwise `"LOCAL"`)
 * `clustermq.host` - The name of the node or device for constructing the
       `ZeroMQ` host address (default is `Sys.info()["nodename"]`)
+* `clustermq.port.range` - A port range used by `clustermq` to initiate 
+      connections. Must be in the form of `lower_port:upper_port`.
```

Comment:
There is no reason to state that the range must be specificed with `:`? `seq` would work as well

Reply:
good point - I removed this sentence altogether.